### ISSUE TITLE
Make v1 the storage version for all sources (except CloudBuildSource)

### ISSUE THREAD

**Issue by danyinggu** [google/knative-gcp#1482]
**Problem**
We will not make v1 the storage version for any of the source in 0.17 cut. We will v1 the storage version in 0.18, create a stand-alone issue to track this.
Relates to #1409, #1410, #1411 and #1412.

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?

**Exit Criteria**
v1 is the storage version for all sources (except CloudBuildSource).

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve?

**Additional context (optional)**
Add any other context about the feature request here.

Cc: @nachocano @capri-xiyue

**Comment by nachocano** [comment]
This should be done after the 0.17 cut

**Comment by capri-xiyue** [comment]
Maybe we should follow knative eventing to have a migration tool.

**Comment by nachocano** [comment]
Yes, we should have a migration tool as eventing is doing. The migration will also help with the deletion of v1alpha1, which we will do in https://github.com/google/knative-gcp/issues/1544

**Comment by nachocano** [comment]
@danyinggu expressed interest in this. If she has cycles later in the sprint, she might be able to pick it up

**Comment by danyinggu** [comment]
@nachocano Is this blocked by #1608 and #1413?

**Comment by capri-xiyue** [comment]
@danyinggu I don't think it's blocked since for this issue, we make Make v1 the storage version for all sources EXCEPT CloudBuildSource)

**Comment by danyinggu** [comment]
> @danyinggu I don't think it's blocked since for this issue, we make Make v1 the storage version for all sources EXCEPT CloudBuildSource)

@nachocano @capri-xiyue Yes, when I created the issue, we decided to promote all sources to v1 except CloudBuildSource, that was why I marked it as "Except CloudBuildSource" here, but I remembered in the last triage or sprint planning, Nacho mentioned that we might want to flip bits for all sources at once, so I'm wondering if this is still the case, then the title of this issue should remove the "Except CloudBuildSource"

**Comment by capri-xiyue** [comment]
@danyinggu  CloudBuildSource will be promoted to v1 in 0.18 release. We need to wait for one release to flip the storage version to v1 which will be 0.19 release. So the title of this issue is still correct.

**Comment by danyinggu** [comment]
@capri-xiyue I created a separate issue for CloudBuildSouce. https://github.com/google/knative-gcp/issues/1610
